From 97728b1371d39f5ddd31004db0ba87b268308c5d Mon Sep 17 00:00:00 2001
From: Mohammed Gamal <mgamal@redhat.com>
Date: Wed, 3 Aug 2022 13:39:56 +0200
Subject: [PATCH 2/2] Update Log Collector default in Comments and Readme
 (#2608)

RH-Author: Mohamed Gamal Morsy <mmorsy@redhat.com>
RH-MergeRequest: 9: Update Log Collector default in Comments and Readme
RH-Commit: [1/1] ff82c0e933649ccd06d3cc48753324cddef2c56a
RH-Bugzilla: 2114824
RH-Acked-by: Vitaly Kuznetsov <vkuznets@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2114824

(cherry picked from commit a560872bec103523e73ddcb5fb5705896545ffce)

Signed-off-by: Mohammed Gamal <mgamal@redhat.com>
---
 README.md                    | 2 +-
 config/debian/waagent.conf   | 2 +-
 config/suse/waagent.conf     | 2 +-
 config/ubuntu/waagent.conf   | 2 +-
 config/waagent.conf          | 2 +-
 tests/data/test_waagent.conf | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/README.md b/README.md
index c69f43e6..f2f3db46 100644
--- a/README.md
+++ b/README.md
@@ -465,7 +465,7 @@ _Default: y_
 
 If set, agent logs will be periodically collected and uploaded to a secure location for improved supportability.
 
-NOTE: This feature is only supported ubuntu 16.04+; this flag will not take effect on any other distro.
+NOTE: This feature relies on the agent's resource usage features (cgroups); this flag will not take effect on any distro not supported.
 
 #### __Logs.CollectPeriod__
 
diff --git a/config/debian/waagent.conf b/config/debian/waagent.conf
index 6bc36604..dfd7afcd 100644
--- a/config/debian/waagent.conf
+++ b/config/debian/waagent.conf
@@ -63,7 +63,7 @@ Logs.Verbose=n
 # Enable Console logging, default is y
 # Logs.Console=y
 
-# Enable periodic log collection, default is n
+# Enable periodic log collection, default is y
 Logs.Collect=y
 
 # How frequently to collect logs, default is each hour
diff --git a/config/suse/waagent.conf b/config/suse/waagent.conf
index ac9f11a2..c617f9af 100644
--- a/config/suse/waagent.conf
+++ b/config/suse/waagent.conf
@@ -66,7 +66,7 @@ Logs.Verbose=n
 # Enable Console logging, default is y
 # Logs.Console=y
 
-# Enable periodic log collection, default is n
+# Enable periodic log collection, default is y
 Logs.Collect=y
 
 # How frequently to collect logs, default is each hour
diff --git a/config/ubuntu/waagent.conf b/config/ubuntu/waagent.conf
index 63635a81..19b56bae 100644
--- a/config/ubuntu/waagent.conf
+++ b/config/ubuntu/waagent.conf
@@ -66,7 +66,7 @@ Logs.Verbose=n
 # Enable Console logging, default is y
 # Logs.Console=y
 
-# Enable periodic log collection, default is n
+# Enable periodic log collection, default is y
 Logs.Collect=y
 
 # How frequently to collect logs, default is each hour
diff --git a/config/waagent.conf b/config/waagent.conf
index c7d10e2f..14cd01c8 100644
--- a/config/waagent.conf
+++ b/config/waagent.conf
@@ -72,7 +72,7 @@ Logs.Verbose=n
 # Enable Console logging, default is y
 # Logs.Console=y
 
-# Enable periodic log collection, default is n
+# Enable periodic log collection, default is y
 Logs.Collect=y
 
 # How frequently to collect logs, default is each hour
diff --git a/tests/data/test_waagent.conf b/tests/data/test_waagent.conf
index a386228a..cc60886e 100644
--- a/tests/data/test_waagent.conf
+++ b/tests/data/test_waagent.conf
@@ -67,7 +67,7 @@ ResourceDisk.MountOptions=None
 # Enable verbose logging (y|n)
 Logs.Verbose=n
 
-# Enable periodic log collection, default is n
+# Enable periodic log collection, default is y
 Logs.Collect=y
 
 # How frequently to collect logs, default is each hour
-- 
2.31.1

